src/platform/brand-consolidation/constants/redirects.json

```diff
@@ -0,0 +1,126 @@
+[
```

Comment:
I thought for awhile about that, and at this point because we have to wrap link tags in the React component designed here, adding redirects will have to be coordinated between a content editor and the engineer. I do like the idea of having it regenerated at build-time so if front-matter adds some redirect data then we can see those changes appear in the diff, but then we also have to keep in mind that the content will eventually be separate from the app code, so then we'd have to do a handoff from the content to the app code through the build-settings. But even then, an engineer will need to implement the redirects by wrapping the updated links.

Really complicated, and I tried to keep it simple to avoid any of the unknowns that may come out of a bigger approach.